pages/home/glossary.mdx

```diff
@@ -0,0 +1,145 @@
+
+Arcade Tools can be presented as an MCP server (via the Arcade Engine), allowing any LLM that supports MCP to access your tools.  In the future, remote MCP servers will be able to be added to your project as additional workers, making their tools available to your agents.
+
+*Learn more about [MCP and Arcade](/home/mcp-overview).*
+
+### OXP
+
+OXP is Arcade's proprietary protocol for tool definition and tool calling that enables AI/LLM agents to access tools and services.  OXP is a more secure and performant alternative to MCP, and is the preferred way to access tools from your agents.
+
+
+## The Arcade Platform
+
+```mermaid
+graph TD
+    A[Tenant] --> B[Project]
+    B --> C[API Key]
+    C --> D[Agent]
+    Z[Account] --> B
+    Z[Account] --> A
+```
+
+### Account
+
+The 'account' is you, the user of Arcade who is developing an application or agent.  You can sign into the Arcade dashboard, manage projects, and more.
+
+### Tenant
+
+A 'tenant' is a collection of projects and users.  It is the top-level organization in Arcade.  You can be a member of one or more tenants, and each tenant can have multiple projects.  Billing is handled at the tenant level.
+
+### Project
+
+A 'project' is a collection of agents and tools.  It is the lowest level of organization in Arcade.  You can have multiple projects, and each project can have multiple agents and tools.  Different Accounts can be members of different projects, and each project will have different API keys.
+
+### API Key
+
+An 'API key' is a secret key that is used to authenticate requests to the Arcade API.  It is used to identify the project that the request is for.  API keys are project-specific.
+
+## Authentication and Billing
+
+```mermaid
+graph TD
+    A[Arcade] --> B[Agent]
+    B --> C[User]
+```
+
+### User
+
+A 'user' is your end-user, the person who is using your application or agent.  Users are counted by the unique `user_id` properties sent when calling tools.  `user_id` values are commonly email addresses, but can be any string or number.
+
+### Monthly Active Users (MAU)
+
+Monthly Active Users are the unique end-users (counted by `user_id`) who have executed a tool in your app/agent within the past month.  If the same `user_id` calls a tool multiple times in the same month, or executes multiple tools, it is only counted once.
+
+### Authentication Starts (Auth Starts)
+
+Authentication Starts are the number of unique users (by `user_id`) who have started the authentication flow for a specific toolkit.  This is a proxy for the number of new users who are using your agent.  The same user authenticating to multiple toolkits will be counted multiple times (e.g. once for Slack and once for Google).
```

Comment:
Since multiple toolkit can use the same auth provider (think Gmail, GoogleCalendar, etc.)
```suggestion
Authentication Starts are the number of unique users (by `user_id`) who have started the authentication flow for a specific auth provider.  This is a proxy for the number of new users who are using your agent.  The same user authenticating to multiple toolkits will be counted multiple times (e.g. once for Slack and once for Google).
```

Reply:
Also a question for myself - If my user "auth starts" for Gmail's `read` scope, and then later "auth starts" for Gmail's `write` scope, then is this counted as 1 or 2 auth starts?

Reply:
Clarified via d28d5fdda4a0ba52031c481e9eda23676db6864b